### ISSUE TITLE
Should allow installation of GS/OS 6.0.3

### ISSUE THREAD

**Issue by knghtbrd** [RasppleII/a2server#2]
(For completeness)

Currently OS/OS 6.0.1 is installed by these scripts. That's appropriate as it is the latest/final release made by Apple, however 6.0.1 Contains several serious bugs including some that corrupt data, and these are unpatched. The Apple // community has obtained source code and released two updated versions. As of this writing, 6.0.3 is current. A2SERVER should offer to install the latest community-released OS version as a help to new users.

**Comment by IvanExpert** [comment]
Agreed, though I personally would like to see that anything past 6.0.1 clearly indicated as a community release, to clearly distinguish it from an official release (especially since there's hardly unity in the community about what should be in these releases, whether they in fact succeed each other or should be considered independent forks, or if they should exist at all).

**Comment by knghtbrd** [comment]
Agreed, and 6.0.1 should always be an option, even if something more 
current becomes the recommended choice due to bug fixes.

Joseph

**Comment by knghtbrd** [comment]
I've slowly begun working on an experimental Python-based replacement for the NetBoot script, you can find that in experimental.  It's a long way from finished and doesn't install 6.0.1 yet, let alone 6.0.3. Nor is it yet "good Python", in part because I'm working from a shell script, my Python is rusty (haven't written much in it since 2.5 days!), and I kind of have bigger goals in mind before this is "done".

That last part is dangerous, but I'll try to keep feeping creaturism somewhat under control, by limiting my focus to:
- Obtaining boot blocks, patching them, and shoving them in a directory
- Extracting the GS/OS disk images
- Running the GS/OS installer scripts ourselves

The shell script also installs tools, utilities, etc. I don't have a problem writing a tool to build a workable default system by installing a useful set of default tools. But I believe in the UNIX philosophy of doing one thing and doing it well. In fact, I'd say the 6502 family strongly reinforce doing things that way by necessity.

In other words, this first. :) Python makes modularity easy, if you want it to write stuff so you can reuse the pieces.

**Comment by knghtbrd** [comment]
Hiding on page 2, this is resolved now!
